### DISCUSSION TITLE
Can not switch users using Auth.js in a Nextjs project with the keycloak provider

### DISCUSSION THREAD

**Discussion by tkowalski-exxeta** [nextauthjs/next-auth#10839] [Help]
Hi everyone,
I am using Auth.js in a Nextjs project (Monorepo using API-Router) with the keycloak provider. Then I sign in the first time everything works as expected. After signing out and logging in again however I am not redirected to the login-page of my oauth-provider but I am logged in directly with the same user as in the first login. 
I want to always navigate to keycloak's login page however to be able to change the user.

What do I need to configure to make that possible?

**Comment by benmarte** [comment]
You need to clear you cookies and local storage or you can try setting the keycloak token expiration to a very low value. Even then it might still use the session info from the browser to refresh the token and log you in.

**Comment by Nekketsu-GIT** [comment]
Hi any update on the issue ? i am stuggling with same issue
